frontend/src/cljs/imas_seamap/map/events.cljs

```diff
@@ -45,6 +45,18 @@
      :east (+ lng (/ img-x-bounds 2))
      :west (- lng (/ img-x-bounds 2))}))
 
+(defn bounds-for-point
+  "Uses current bounds and a map point coordinate to get the map bounds centered
+   on that point."
+  [{:keys [lat lng]}
+   {:keys [north south east west]}]
+  (let [img-x-bounds (Math/abs (- west east))
```

Comment:
I'm not super fussed, but "img" tends to have connotations of pixels rather than coordinates, in our context.

Reply:
(cf getfeatureinfo, where you have both geographic bounds and x/y pixel coords)